src/Raven.Server/Documents/Indexes/Persistence/Lucene/Documents/LuceneDocumentConverterBase.cs

```diff
@@ -315,6 +317,14 @@ int HandleArray(IEnumerable itemsToIndex)
                 return newFields;
             }
 
+            if (valueType == ValueType.DynamicDictionary)
+            {
+                instance.Add(GetOrCreateField(path + IsDictionaryFieldSuffix, TrueString, null, null, storage, Field.Index.NOT_ANALYZED_NO_NORMS, Field.TermVector.NO));
+                newFields++;
+
+                valueType = ValueType.ConvertToJson;
```

Comment:
So we're intentionally changing it to `ValueType.ConvertToJson` ?

Reply:
I really have some doubts if we need to introduce any new concept to LuceneDocumentCoverter? Since we're indexing it same as we did it - `ConvertToJson`

Reply:
Yes, it is the same as it was.
I introduced _ValueType.DynamicDictionary_ just to get the ability to add __IsDictionary_ suffix, I tried to follow the same logic we have with Array (we adding this field there also). Anyway I can easily remove it, if needed

Reply:
I'm think sure we don't need to change anything in `LuceneDocumentConverterBase` because from the Lucene point of view it doesn't matter. We're just storing a dictionary as JSON and that's it - so same behavior as we had.

@ppekrol thoughts?

Reply:
I've talked to Pawel. Given that we don't have any requirement to introduce _IsDictionary at this let's drop it.

Reply:
done

Reply:
"done"? - did you forget to push? :)